tests/test_modeling_common.py

```diff
@@ -201,12 +199,6 @@ def test_torchscript(self):
 
         self._create_and_check_torchscript(config, inputs_dict)
 
-    def test_torchscript_output_attentions(self):
```

Comment:
Although it never looks good to delete a test :D, I think it's fine here:

- `self._create_and_check_torchscript` is already tested with `input_dict["output_attentions"] = True` in the next test, so this test does not create any additional test coverage in my opinion.